etc/apache2/vhosts.d/openqa-common.inc

```diff
@@ -29,6 +42,7 @@ ProxyPreserveHost On
 ProxyPass /robots.txt !
 ProxyPass /favicon.ico !
 ProxyPass /images !
+ProxyPass /image !
```

Comment:
Ah, the route is there primarily for url_for and non-apache setups, ok, get it now.